weblate/trans/models/component.py

```diff
@@ -3162,6 +3181,12 @@ def after_save(
             was_change = self.create_translations(
                 force=True, changed_template=changed_template, run_async=True
             )
+            # This fires if a pattern exist and the file format is monolingual
+            with transaction.atomic():
+                translations = Translation.objects.filter(component=self)
+                for translation in translations:
+                    translation.check_sync(force=True)
```

Comment:
The `create_translations` above does pretty much this, this block should not be needed at all.

Reply:
Used the `create_translations`.